packages/ketcher-react/src/script/editor/tool/rotate-controller.ts

```diff
@@ -27,11 +32,11 @@ const RIGHT_ARROW_PATH =
 
 class RotateController {
   isRotating!: boolean
+  handleCenter!: Vec2
```

Comment:
is it still required to be public? or it can be 'private'

Reply:
My bad, ignored it